### ISSUE TITLE
Building 2.2.0 fails at cppo

### ISSUE THREAD

**Issue by tuncer** [ocaml/opam#3718]
Building 2.0.2 fails for me as follows:

```
$ ocaml -version
The OCaml toplevel, version 4.07.1
$ dune --version
1.6.2

$ git clean -fxd
$ git checkout 2.0.2
$ ./configure
$ make lib-ext
$ make
[...]
dune build --profile=release  opam-installer.install opam.install
File "src_ext/cppo/src/jbuild", line 10, characters 10-50:
10 |     (echo "let cppo_version = \"${version:cppo}\"")))))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Error: Package "cppo" doesn't exist in the current project.
make: *** [Makefile:107: opam.install] Error 1
```

The OCaml switch in use is 4.07.1+flambda.

Maybe a dune file issue?

**Comment by dra27** [comment]
It looks like it might be a Dune issue - you could try downgrading Dune to earlier versions (I’ll get to it, but probably not until the new year)

**Comment by avsm** [comment]
You need dune 1.5.0 or dune master I think

**Comment by dra27** [comment]
This issue was fixed in Dune 1.6.3

**Comment by rjbou** [comment]
#3887 & #3889 merged. Thanks!
